### PR TITLE
fix #74255 uniqueten.net

### LINKED ISSUE #74255: uniqueten.net
### Issue URL (Ads)
 [https://uniqueten.net/importance-of-customer-loyalty-in-e-commerce-industry/#wpsafegenerate](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Funiqueten.net%2Fimportance-of-customer-loyalty-in-e-commerce-industry%2F%23wpsafegenerate)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 15
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List
Custom filter subscriptions: | https://raw.githubusercontent.com/AdguardTeam/FiltersRegistry/master/filters/filter_15_DnsFilter/filter.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#74255

### CODE REVIEW

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -242,6 +242,8 @@ artipedia.id#%#//scriptlet("prevent-window-open")
 artipedia.id#%#//scriptlet('remove-class', 'get-link', 'a[class="btn btn-success btn-lg get-link"][target="_blank"]')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/74297
 elil.cc#%#//scriptlet('remove-attr', 'onmouseover|onclick|onmouseout', '.save-btn.pull-right')
+! https://github.com/AdguardTeam/AdguardFilters/issues/74255
+uniqueten.net,ultraten.net#%#//scriptlet("adjust-setInterval", "wpsafe-generate")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:246]: This rule should be in `tweaks` section.
`uniqueten.net,ultraten.net#%#//scriptlet("adjust-setInterval", "wpsafe", "", "0.02")`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt]: Done.

File: AnnoyancesFilter/sections/tweaks.txt [hunk 2]
```diff
@@ -2,6 +2,8 @@
 !######### CSS tweaks #########
 !##############################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74255
+uniqueten.net,ultraten.net#%#//scriptlet("adjust-setInterval", "wpsafe", "", "0.02")
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:6]: As you can see, it's a `CSS tweaks`. `JS tweaks` are located below.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/tweaks.txt (hunk 2 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/tweaks.txt]: Done.

**Review by zloyden — APPROVED** [review]

**Pull request merged**